kustomize-fns/image-prefix/function.go

```diff
@@ -99,8 +98,6 @@ func (f *ImagePrefixFunction) Filter(inputs []*yaml.RNode) ([]*yaml.RNode, error
 // replaceImage looks for an annotation changing docker image prefixes and if it is present
 // applies it to all the images.
 func (f *ImagePrefixFunction) replaceImage(r *yaml.RNode) error {
-	m , _  := r.GetMeta()
```

Comment:
How is this change related to this PR?

Reply:
Miscellaneous cleanup. Not directly related.